Misc/NEWS.d/next/Library/2025-03-06-15-56-12.gh-issue-130925.USr9bm.rst

```diff
@@ -0,0 +1,3 @@
+:func:`asyncio.StreamReader.close` now exists so that it's possible to
+signal to sub-processes executed via :func:`asyncio.create_subprocess_exec`
+that they may cease generating output and exit cleanly.
```

Comment:
```suggestion
Add :func:`asyncio.StreamReader.close` method. so It can signal to sub-processes executed via :func:`asyncio.create_subprocess_exec`
that they may cease generating output and exit cleanly.
```

Reply:
I think the "so" was a typo? Otherwise I can change this